src/docker/compose/index.go

```diff
@@ -12,44 +11,30 @@ func BuildImage(opts ImageOptions) error {
 
 // BuildAllImages builds all the docker images defined in docker-compose.yml
 func BuildAllImages(opts BaseOptions) error {
-	return util.RunAndLog(opts.DockerComposeDir, opts.Env, opts.Logger, "docker-compose", "build")
-}
-
-// CreateNewContainer creates a new docker container for the given service
-func CreateNewContainer(opts ImageOptions) error {
-	return util.RunAndLog(opts.DockerComposeDir, opts.Env, opts.Logger, "docker-compose", "create", "--build", opts.ImageName)
+	return util.RunAndPipe(opts.DockerComposeDir, opts.Env, opts.Writer, "docker-compose", "build")
 }
 
 // KillAllContainers kills all the containers
 func KillAllContainers(opts BaseOptions) error {
 	return util.RunAndPipe(opts.DockerComposeDir, opts.Env, opts.Writer, "docker-compose", "down")
 }
 
-// KillContainer kills the docker container of the given service
-func KillContainer(opts ImageOptions) error {
```

Comment:
This was dead code that was never being called. Does our linter not catch things like this?

Reply:
Its a public function so that might be a function we as a library are exporting. If it was private and unused the linter would catch it